web/src/settings_config.ts

```diff
@@ -839,7 +839,7 @@ export const all_notifications = (settings_object: Settings): AllNotifications =
             ),
         },
         {
-            label: $t({defaultMessage: "DMs, mentions, and alerts"}),
+            label: $t({defaultMessage: "DMs, mentions, and watched notifications"}),
```

Comment:
Currently renamed "alerts" to "watched notifications". Need suggestions on naming this.

Reply:
"DMs and mentions" is probably the best we can do for now. I think it's currently not the case that they can send email/push notifications anyway (#28779, etc.), so not mentioning those is probably an improvement.

You should do a clean commit series for this -- maybe a single prep commit that changes this string (`git grep 'DMs, mentions'`) across the app, including the Help Center, and then a next commit that's the "watched phrases" transition.